cds_ils/importer/providers/springer/rules/document.py

```diff
@@ -298,7 +298,7 @@ def id_isbns(self, key, value):
     """Translate identifiers isbn."""
     _identifiers = self.get("identifiers", [])
 
-    isbn_value = clean_val("a", value, str)
+    isbn_value = clean_val("a", value, str) or (clean_val('z', value, str))
```

Comment:
if both subfields will be missing, we will have isbn_value = False

Reply:
it will just be skipped in the next if statement, that is correct behaviour right?